docker/Dockerfile

```diff
@@ -33,4 +33,4 @@ VOLUME /var/lib/postgres
 VOLUME /var/spool/faf
 
 EXPOSE 80
-CMD ["sh", "-c", "crond && apachectl -D FOREGROUND"]
+CMD ["sh", "-c", "crond && init_db && apachectl -D FOREGROUND"]
```

Comment:
Wouldn't it be better to create just one init script and include all these into it?

Reply:
Great idea! See my new commit in this PR.